### PR TITLE
feat(options): customSVGIcons API 变更

### PR DESCRIPTION
**Pull request by lijinke666**
### 👀 PR includes

✨ Feature

- [x] New feature

### 📝 Description

1. BREAKING CHANGE: svg 变更为 src

```diff
const s2Options = {
  customSVGIcons: [
    {
      name: 'CustomIcon',
-     svg: 'https://gw.alipayobjects.com/zos/bmw-prod/f44eb1f5-7cea-45df-875e-76e825a6e0ab.svg',
+     src: 'https://gw.alipayobjects.com/zos/bmw-prod/f44eb1f5-7cea-45df-875e-76e825a6e0ab.svg',
    },
  ]
}
```

2. 内置 icon 格式优化

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/common/interface/basic.ts [hunk 1]
```diff
@@ -263,14 +263,14 @@ export interface CustomSVGIcon {
 
    import Icon from 'path/to/xxx.svg'
```
Comment (wjgogogo) [inline packages/s2-core/src/common/interface/basic.ts:264]: 是继续叫 CustomSVGIcon 还是改成 CustomIcon 就行了

Comment (lijinke666) [reply packages/s2-core/src/common/interface/basic.ts:264]: 想了下, 还是叫前者, 虽然 png 之类的也支持, 但是默认都是按照 svg 处理的, 会替换颜色, 内置的也全部是 svg

https://github.com/antvis/S2/blob/f70c788afa61e8e107fe6d6975375d663a325e09/packages/s2-core/src/common/icons/gui-icon.ts#L89-L91

**Review by wjgogogo — APPROVED** [review]

**Pull request merged**